dolphinscheduler-tools/src/main/resources/application.yaml

```diff
@@ -41,6 +41,8 @@ spring:
   config:
     activate:
       on-profile: mysql
+      # TODO Add this to test check style
+      driver-class-name: com.mysql.jdbc.Driver
```

Comment:
I add this line to test whether check style could find it and raise error, but failed. Did I miss something?

Reply:
It seems that the check style will not effect config file.

Reply:
I found out it have parameter in https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#resourceIncludes could check file in resource dir. I change the file ext from `properties` to `yaml` but I still failed